crates/matrix-sdk/src/client/builder.rs

```diff
@@ -132,7 +132,7 @@ impl ClientBuilder {
     ///
     /// This is a shorthand for
     /// <code>.[store_config](Self::store_config)([matrix_sdk_indexeddb]::[make_store_config](matrix_sdk_indexeddb::make_store_config)(path, passphrase).await?)</code>.
-    #[cfg(feature = "indexeddb")]
+    #[cfg(all(feature = "indexeddb", target_arch = "wasm32"))]
```

Comment:
This will make them disappear from docs. What was clippy complaining about?

Reply:
that returning `OpenStoreError` doesn't exist, which is only exposed by `indexeddb` when under `wasm32` arch ... I honestly don't understand why that would have ever passed...

Reply:
I see. I think this should fix it: https://github.com/matrix-org/matrix-rust-sdk/pull/910